pkg/addons2/addons.go

```diff
@@ -44,6 +46,16 @@ func InstallAddons(ctx context.Context, opts InstallOptions) error {
 		},
 	}
 
+	drEnabled, err := helpers.DisasterRecoveryEnabled(opts.License)
```

Comment:
i think it would be cleaner to have a way for opts.IsDisasterRecoveryEnabled as a variable instead of this lookup here. Let's aim to make it more self contained.

Reply:
hmm, this is just using `license.Spec.IsDisasterRecoverySupported`, and the license is an input to this function. i can just get rid of this helper function

Reply:
updated. let me know if that works